swlor2_2da/feat.2da

```diff
@@ -1888,3 +1888,4 @@
 1884   FlameBlast                                       16860319   16860320      ife_x3_ptool01     ****             ****     ****     ****     ****     ****     ****     ****          ****          ****          0              0              1                  ****       ****    ****      ****        ****      ****         ****         1            ****         ****         ****         ****         ****         ****       ****               ****        ****                ****                                             6                 ****          99         ****            ****       ****          0            0           
 1885   Infusion1                                        16860327   16860328      ife_infusion       ****             ****     ****     ****     ****     ****     ****     ****          ****          ****          0              0              1                  ****       ****    ****      ****        ****      ****         ****         ****         ****         ****         ****         ****         ****         ****       ****               ****        ****                ****                                             6                 ****          99         ****            ****       ****          0            0           
 1886   Infusion2                                        16860329   16860330      ife_infusion       ****             ****     ****     ****     ****     ****     ****     ****          ****          ****          0              0              1                  ****       ****    ****      ****        ****      ****         ****         ****         ****         ****         ****         ****         ****         ****       ****               ****        ****                ****                                             6                 ****          99         ****            ****       ****          0            0           
+1887   ZenMarksmanship                                  16860331   16860332      ife_X1ZenArc       ****             ****     ****     ****     ****     ****     ****     ****          ****          ****          0              0              1                  ****       ****    ****      ****        ****      ****         ****         ****         ****         ****         ****         ****         ****         ****       ****               ****        ****                ****                                             6                 ****          99         ****            ****       ****          0            0           
```

Comment:
This is going to conflict with the incoming Beast Mastery perks, sorry. Please reassign this to 1995, leaving gaps in between in the file for 1887-1994  (remember to **** all of the fields for each line).

Additionally, you need to add this entry to cls_feat_fight.2da

Reply:
Done.